craft_parts/executor/environment.py

```diff
@@ -170,9 +173,6 @@ def _get_global_environment(info: ProjectInfo) -> dict[str, str]:
     if Features().enable_partitions:
         global_environment.update(_get_environment_for_partitions(info))
 
-    global_environment["CRAFT_STAGE"] = str(info.stage_dir)
-    global_environment["CRAFT_PRIME"] = str(info.prime_dir)
-
```

Comment:
this was probably here due to an ordered dict thing, reverting this, means we essentially revert this file back to its original state